features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/defaultTheme.js

```diff
@@ -179,9 +179,9 @@ export default {
         },
         // lifeCycleImage: '/site/public/images/logo.svg',
         // Uncomment above if you want to add a custom image to the lifecycle diagram.
-        adminRole: 'admin@carbon.super',
+        adminRole: 'admin',
```

Comment:
This is actually the admin username, instead of this we could keep the scope name just in case someone wants to change it

`apim:admin` and check for this scope in the [above code](https://github.com/wso2/carbon-apimgt/pull/10171/files#diff-5e546fe7454618eebd97dc656f40127ed039d9b398ba478196796dc1ac7b5798R158) to check for admin permissions 
@malinthaprasan WDYT ?

Reply:
Fixed